docs/getting-started-with-scalardb-on-cosmosdb.md

```diff
@@ -37,14 +37,14 @@ The **scalardb.properties** (getting-started/scalardb.properties) file holds the
     
 ```
 # Comma separated contact points
-scalar.db.contact_points=<YOUR_ACCOUNT_URI>
+scalar.db.contact_points=<COSMOS_DB_ACCOUNT_URI>
 
 # Port number for all the contact points. Default port number for each database is used if empty.
 #scalar.db.contact_port=
 
 # Credential information to access the database
 scalar.db.username=
-scalar.db.password=<YOUR_ACCOUNT_PASSWORD>
+scalar.db.password=<COSMOS_DB_PRIMARY_KEY>
```

Comment:
It might be too specific because not only the primary key but also the secondary key can be used.

Reply:
I think Yuji's suggestion is reasonable, so why don't you apply it and leave something like this as a note before `Please follow ...` sentence.

```
Note that you can use a primary key or a secondary key for `<COSMOS_DB_KEY>`.
```

Reply:
@yito88 @feeblefakie san,
Thank you for your suggestions.
Document updated accordingly.